src/felles-komponenter/EkspanderbartTekstomrade.tsx

```diff
@@ -0,0 +1,45 @@
+import { NedChevron, OppChevron } from 'nav-frontend-chevron';
+import React, { useState } from 'react';
+
+import styles from './EkspanderbartTekstomrade.module.less';
+
+const MoreOrLess = ({ text, maxCharacters }: { text: string; maxCharacters: number }) => {
+    const [hasLongText, setHasLongText] = useState(true);
+    const toggleMoreOrLess = () => {
+        setHasLongText(!hasLongText);
+    };
+    if (text.length > maxCharacters) {
+        return (
+            <p>
+                {hasLongText ? text.slice(0, maxCharacters) + ' ... ' : text + ' '}
+                <span className={styles.nobreak}>
+                    <span className={styles.color}>
+                        <span onClick={toggleMoreOrLess} className={styles.padding}>
+                            {hasLongText ? 'Les mer' : 'Vis mindre'}
+                        </span>
+                        {hasLongText ? <NedChevron /> : <OppChevron />}
+                    </span>
+                </span>
+            </p>
+        );
+    } else {
+        return <p>{text}</p>;
+    }
```

Comment:
Her kan du droppe else og heller gjøre slik:
```suggestion
      return <p>{text}</p>;
```